operator/Makefile

```
@@ -291,8 +299,12 @@ test-upgrade: kuttl bundle-post-process ## Run OLM-based operator upgrade tests.
 .PHONY: test-bundle-helpers
 test-bundle-helpers: ## Run Python unit tests against helper scripts.
```

checkmake (0.3.2)

[warning] Target body for "test-bundle-helpers" exceeds allowed length of 5 lines (7).

(maxbodylength)